OpenRA.Mods.Common/Scripting/Properties/TransportProperties.cs

```diff
@@ -31,6 +31,9 @@ public TransportProperties(ScriptContext context, Actor self)
 		[Desc("Specifies whether transport has any passengers.")]
 		public bool HasPassengers { get { return cargo.Passengers.Any(); } }
 
+		[Desc("Specifies the amount of passengers.")]
+		public int Passengers { get { return cargo.Passengers.Count(); } }
```

Comment:
This should be `PassengerCount`, just `Passengers` would make me think this is a table of actors.